db/db-connect.go

```diff
@@ -0,0 +1,78 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func InitDB() (string, error) {
+	appPath := os.Getenv("TODO_DBFILE")
+
+	if appPath == "" {
+		nowPath, err := os.Getwd()
+		if err != nil {
+			fmt.Println("Error getting current working directory")
+			return "", err
```

Comment:
Лучше везде оборачивать ошибки
```suggestion
			return "", fmt.Errorf("error getting current working directory: %w", err)
```